cogs/currencycog.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import discord
+from discord.ext import commands
+
+
+class CurrencyCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def cconvert(self, ctx, *args):
+        """Converts between currencies"""
+
+        # To do:
+        # 1. make it output a fancy embed thing
+        # 2. round the output (number of decimal places differs by currency)
+
+        try:
+            initial_value = float(args[2]) if len(args) > 2 else 1
+            payload = {"symbols": args[1], "base": args[0]}
+            r = requests.get("https://api.exchangeratesapi.io/latest", params=payload)
+            apidata = r.json()
+            await ctx.send(float(apidata["rates"][args[1]])*initial_value)
+        except IndexError:
```

Comment:
You won't need to check for index errors after changing to explicitly named arguments :+1: